authorization_services/topics/policy-group-policy.adoc

```diff
@@ -1,12 +1,12 @@
 [[_policy_group]]
-= Group-Based Policy
+= Group-based policy
 
 You can use this type of policy to define conditions for your permissions where a set of one or more groups (and their hierarchies) is permitted to access an object.
 
-To create a new group-based policy, select *Group* in the dropdown list in the upper right corner of the policy listing.
+To create a new group-based policy, select *Group* in the item list in the upper right corner of the policy listing.
 
-.Add Group-Based Policy
-image:{project_images}/policy/create-group.png[alt="Add Group-Based Policy"]
+Group-Based Policy
```

Comment:
Image title casing

Reply:
I think this one should remain initial capitals to match the UI.